client/state/selectors/is-woocommerce-core-profiler-flow.ts

```diff
@@ -11,9 +11,10 @@ import type { AppState } from 'calypso/types';
  * @returns {?boolean}        Whether the user reached Calypso via the WooCommerce Core Profiler flow
  */
 export const isWooCommerceCoreProfilerFlow = ( state: AppState ): boolean => {
+	const allowed_from = [ 'woocommerce-core-profiler', 'woocommerce-payments' ];
```

Comment:
small nit but I think `snake_case` isn't conventional within wp-calypso for variable names

Reply:
Thank you! @rjchow 

Old habit 🥲